docs/api.md

```diff
@@ -2513,6 +2514,9 @@ This method will throw if the response body is not parsable via `JSON.parse`.
 
 Contains a boolean stating whether the response was successful (status in the range 200-299) or not.
 
+#### response.remoteAddress()
+- returns: <[string]> The formatted address and port of the remote server.
```

Comment:
I'd return IP address and port separately: it's a little error-prone to parse out port number with IPv6 addresses.

For example, this method can return an object `{ip, port}`